Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -1509,7 +1509,10 @@ - (nullable UIViewController *)previewingContext:(id<UIViewControllerPreviewing>
             UIView *view = [sideScrollingCell viewForSubItemAtIndex:index];
             CGRect sourceRect = [view convertRect:view.bounds toView:self.collectionView];
             previewingContext.sourceRect = sourceRect;
-            NSUInteger indexes[3] = {previewIndexPath.section, previewIndexPath.item, index};
+            
+            NSUInteger articleToPeekIndex = group.articleURLString.integerValue;
```

Comment:
Since `NSUInteger` can never be nil, we might want to use a different value if `articleToPeekIndex == NSNotFound` but not sure what's the best solution in this case